test/resources/examples/portal/src/browser/.generated/clients/UsersClient.ts

```diff
@@ -0,0 +1,51 @@
+//
+// GENERATED SOURCE - DO NOT EDIT
+//
+
+import { RestClient, RestClientRequest } from '@kapeta/sdk-web-rest-client';
+
+export class UsersClient extends RestClient {
+    constructor() {
+        super('api/rest/users');
+    }
+
+    /**
+     * Add user
+     * HTTP: POST /api/rest/users/users/{userId}
```

Comment:
Not caused by this PR, but I notice that the path here is prepended with `/api/rest/` but below it is not. I think it should be without `/api/rest/`.